tic-tac-toe-app/GameCell.jsx

```diff
@@ -0,0 +1,27 @@
+function GameCell({
+  cellIndex,
+  cells,
+  setCells,
+  winner,
+  turn,
+  setTurn,
```

Comment:
Cool use of the destructing assignment within the function definition. This can help keep props management cleaner than accessing each prop within the function definition with something like `props.someProperty`. 👍